src/main/java/at/ac/tuwien/kr/alpha/common/atoms/Literal.java

```diff
@@ -0,0 +1,13 @@
+package at.ac.tuwien.kr.alpha.common.atoms;
+
+/**
+ * Copyright (c) 2017, the Alpha Team.
+ */
+public interface Literal extends Atom {
+
+	boolean isNegated();
+
+	default boolean isPositive() {
```

Comment:
Please remove `isPositive()` ... There should be only one possibility to get the polarity of a literal. `isPositive()` is redundant in a very obvious way, and it will only make reading our code harder (two variants capture the same concept).